### ISSUE TITLE
Configuration history filled up with ngx_autoblock.php changes

### ISSUE THREAD

**Issue by ruffy91** [opnsense/plugins#942]
Every minute there is one such change
```
--- /conf/backup/config-1540844700.5377.xml	2018-10-29 21:25:00.539144000 +0100
+++ /conf/config.xml	2018-10-29 21:26:00.482535000 +0100
@@ -573,7 +573,7 @@
   </widgets>
   <revision>
     <username>(system)</username>
-    <time>1540844640.523</time>
+    <time>1540844760.4583</time>
     <description>/usr/local/opnsense/scripts/nginx/ngx_autoblock.php made changes</description>
   </revision>
   <OPNsense>
```
This means that reverting changes is no longer possible after 60 minutes with the default settings.

**Comment by fabianfrz** [comment]
@fichtner can we disable the history creation for this script or do I have to implement some logic here?

**Comment by fichtner** [comment]
@fabianfrz best to avoid calling write_config() if there are no changes

**Comment by fabianfrz** [comment]
fixed in #930
